### PR TITLE
Fix migrations for sqlsrv

### PR DESCRIPTION
**Pull request by holystix**
Modifies migrations to work with sqlsrv (Microsoft SQL Server) DB connection. Does not break modified migrations for mysql.

Some of the changes may seem weird (such as altering the same table multiple times in a single migration), but that's due to the fact that sqlsrv is not well supported by Laravel itself. See https://github.com/laravel/framework/issues/4402#issuecomment-44027350.

### CODE REVIEW

File: database/migrations/2017_06_07_113300_modify_primary_key_in_settings_table.php [hunk 1]
```diff
@@ -10,10 +11,36 @@ class ModifyPrimaryKeyInSettingsTable extends Migration
...
+
+        Schema::table('settings', function (Blueprint $table) {
+            $table->dropColumn('id');
+            $table->string('name')->primary()->change();
+        });
+    }
+
+    public function down()
+    {
+        Schema::table('settings', function (Blueprint $table) {
```
Comment (exabyssus) [inline database/migrations/2017_06_07_113300_modify_primary_key_in_settings_table.php]: Shoudn't these two blocks be combined?

File: database/migrations/2019_01_09_101302_add_activation_columns_to_nodes_table.php [hunk 2]
```diff
@@ -49,4 +53,12 @@ public function down()
             $table->dropColumn(['expire_at', 'activate_at']);
         });
     }
+
+    protected function dropDefaultConstraintForActiveColumn()
```
Comment (exabyssus) [inline database/migrations/2019_01_09_101302_add_activation_columns_to_nodes_table.php]: It's hard to see that this is for mssql. Could you name it accordingly and add PHPDoc?

File: database/migrations/2017_06_07_113300_modify_primary_key_in_settings_table.php (hunk 1 above)
Comment (holystix) [reply database/migrations/2017_06_07_113300_modify_primary_key_in_settings_table.php]: Normally, they should, but sqlsrv doesn't allow for a primary key to be dropped in the same transaction where a new primary key is added.

File: database/migrations/2019_01_09_101302_add_activation_columns_to_nodes_table.php (hunk 2 above)
Comment (holystix) [reply database/migrations/2019_01_09_101302_add_activation_columns_to_nodes_table.php]: Resolved in https://github.com/arbory/arbory/pull/108/commits/3e5ce64057ef95aade5eb168eceabc3c4ba3654b.

**Pull request merged**